stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/builder/components/Field.java

```diff
@@ -141,4 +140,17 @@ private Builder(DescriptorProtos.FieldDescriptorProto fieldDescriptor) {
         FIELD_LABEL_MAP.put(DescriptorProtos.FieldDescriptorProto.Label.LABEL_REQUIRED, null);
         FIELD_LABEL_MAP.put(DescriptorProtos.FieldDescriptorProto.Label.LABEL_REPEATED, "[]");
     }
+
+    private static final String[] RESERVED_LITERAL_NAMES = {
+            "import", "as", "public", "private", "external", "final", "service", "resource", "function", "object",
+            "record", "annotation", "parameter", "transformer", "worker", "listener", "remote", "xmlns", "returns",
+            "version", "channel", "abstract", "client", "const", "typeof", "source", "from", "on", "group", "by",
+            "having", "order", "where", "followed", "for", "window", "every", "within", "snapshot", "inner", "outer",
+            "right", "left", "full", "unidirectional", "forever", "limit", "ascending", "descending", "int", "byte",
+            "float", "decimal", "boolean", "string", "error", "map", "json", "xml", "table", "stream", "any",
+            "typedesc", "type", "future", "anydata", "handle", "var", "new", "__init", "if", "match", "else",
+            "foreach", "while", "continue", "break", "fork", "join", "some", "all", "try", "catch", "finally", "throw",
+            "panic", "trap", "return", "transaction", "abort", "retry", "onretry", "retries", "committed", "aborted",
+            "with", "in", "lock", "untaint", "start", "but", "check", "checkpanic", "primarykey", "is", "flush",
+            "wait", "default"};
```

Comment:
Isn't it difficult to maintain this in the future as we have to change this array whenever new reserved word is adding to the language? There is a possibility to miss this.

Reply:
Sorry for the noise. Just noticed your comment on requeesting an API.